eligibility_api/server.py

```diff
@@ -68,5 +70,5 @@ def make_token(
         "enc": jwe_cek_enc,
     }
     encrypted_token = jwt.JWT(header=header, claims=signed_payload)
-    encrypted_token.make_encrypted_token(client_public_key)
+    encrypted_token.make_encrypted_token(create_jwk(client_public_key))
     return encrypted_token.serialize()
```

Comment:
I know we're not focused on the `server` code just yet, but it feels like most of this `make_token` stuff should go away in favor of the new classes in the `tokens` module right? Can wait for the next set of work on `server`.

Reply:
Yes, I agree wholeheartedly!